interfaces/builtin/kvm.go

```diff
@@ -38,8 +48,60 @@ const kvmConnectedPlugAppArmor = `
 
 var kvmConnectedPlugUDev = []string{`KERNEL=="kvm"`}
 
+type kvmInterface struct {
+	commonInterface
+}
+
+var procCpuinfo = "/proc/cpuinfo"
+
+func getCpuFlags() (flags string, err error) {
+	buf, err := ioutil.ReadFile(procCpuinfo)
+	if err != nil {
+		// if we can't read cpuinfo, we want to know _why_
+		return "", fmt.Errorf("error: %v", err)
+	}
+
+	pattern := []byte("\nflags\t\t:")
```

Comment:
I'm slightly worried about `\t\t`, i.e. this pattern is pretty static, if /proc/flags ever gets something like `cache_alignment_size_offset_table_address : 0xdeadbeef` this may break. As much as I dislike regexp I wonder if they will be more appropriate here so that we can simply ignore the whitespace?

Reply:
I share your discomfort, but I was also avoiding regex :) I'll see what I can do

Reply:
I'm using regexp, is cleaner and not in a hot code path.